src/saveload/saveload.h

```diff
@@ -403,6 +403,8 @@ enum SaveLoadVersion : uint16_t {
 	SLV_FIX_SCC_ENCODED_NEGATIVE,           ///< 353  PR#14049 Fix encoding of negative parameters.
 	SLV_ORDERS_OWNED_BY_ORDERLIST,          ///< 354  PR#13948 Orders stored in OrderList, pool removed.
 
+	SLV_INDUSTRY_ACCEPTED_HISTORY,          ///< 355  PR#14321 Industry acceptance history.
```

Comment:
```suggestion
	SLV_INDUSTRY_ACCEPTED_HISTORY,          ///< 355  PR#14321 Industry history of cargo delivered and waiting.
```

Acceptance history sounds like an industry denying cargo deliveries, as I think ECS does. Better wording?